plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -246,6 +251,14 @@ class Telemetry : public PluginBase {
      */
     Result set_rate_in_air(double rate_hz);
 
+    /**
+     * @brief Set rate of status text updates (synchronous).
+     *
+     * @param rate_hz Rate in Hz.
+     * @return Result of request.
+     */
+    Result set_rate_status_text(double rate_hz);
```

Comment:
I would not implement this. Statustext come when they happen. They don't really have a rate.